### PR TITLE
[guides] migrate border-router guides

### PR DESCRIPTION
**Pull request by gavinmcgovern**

### CODE REVIEW

File: site/en/guides/border-router/docker/cleanup-and-basics.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Use `Ctrl+C` in the terminal window running OTBR Docker to stop the process
+gracefully.
+
+If you are running a simulated RCP, also use `Ctrl+C` to stop the processes for
+`socat` and the RCP node itself.
+
+## Restart OTBR Docker
+
+Follow the same complete procedure in [Run OTBR
+Docker](../../../../guides/border-router/docker/run.md) and [Test
```
Comment (Vyrastas) [inline site/en/guides/border-router/docker/cleanup-and-basics.md:14]: These relative links should move up only 3 directories, not 4.  Anything in the /docker directory should be 3

File: site/en/guides/border-router/raspberry-pi.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+*   A microSD card and microSD card reader ("SD card" in this guide)
+
+Access the RPi with an external monitor (via HDMI) and keyboard (via USB), or
+connect a [serial cable](../../../guides/border-router/serial-cables.md) from the RPi to
+your computer and enable the serial console.
+
+To use RPi with OTBR:
+
+1.  [Download and install the OS](#download-and-install-the-os).
+1.  [Build and install OTBR](../../../guides/border-router/build/index.md).
```
Comment (Vyrastas) [inline site/en/guides/border-router/raspberry-pi.md:21]: The local links for pages in the border-router/ directory should only need to go up 2, not 3

File: site/en/guides/border-router/tools.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+*   Thread Network Name
+
+> Note: The Commissioner Credential is a user-defined string between 6 and 255 characters, UTF-8 encoded.
+  
+### Usage
+
+Syntax:
+
+```
+pskc {&lt;commissioner-credential&gt; &lt;extpanid&gt; &lt;network-name&gt;}
```
Comment (Vyrastas) [inline site/en/guides/border-router/tools.md:30]: Each parameter should be a replaceable variable

pskc {commissioner-credential} {extpanid} {network-name}

File: site/en/guides/border-router/tools.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+*   Joiner ID (EUI-64)
+*   Byte length of steering data (optional, default is 16)
+
+### Usage
+
+Syntax:
+
+```
+steering-data {[length] &lt;joiner-id&gt;}
```
Comment (Vyrastas) [inline site/en/guides/border-router/tools.md:66]: Let's try this

steering-data [{length}] {joiner-id}

File: site/en/guides/border-router/serial-cables.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+both cables, the black wire should be connected to Pin 1 (GND), which is
+closest to the Ethernet port.
+
+> Warning: To avoid hardware damage, always ensure cables are properly connected
+before use.
+
+<figure>
+<a href="../../guides/images/otbr-cables-adafruit.png"><img src="../../guides/images/otbr-cables-adafruit.png" width="600" border="0" alt="BeagleBone Black Adafruit" /></a><figcaption>Adafruit serial cable</figcaption>
+</figure>
+
```
Comment (Vyrastas) [inline site/en/guides/border-router/serial-cables.md:36]: Can you add a <br /> here, it'll help on DevSite

File: site/en/guides/border-router/docker/test-connectivity.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+1.  To get a translated IPv4 address, use a website like
+    [findipv6.com](https://findipv6.com/ipv4-toipv6/).
+
+1.  Translate the IPv4 address you wish to test. For example, `172.217.164.110`
+    translated to IPv6 is `::ffff:acd9:a46e`.
+
+1.  Using only the last 4 bytes of the resulting IPv6 address, combine it with
+    the well-known prefix of `64:ff9b::/64` to get a new IPv6 address:
+    ```64:ff9b::acd9:a46e```
```
Comment (Vyrastas) [inline site/en/guides/border-router/docker/test-connectivity.md:139]: Format this like the one in the step below

```
64:ff9b::acd9:a46e
```

**Review by Vyrastas — CHANGES_REQUESTED** [review]

File: site/en/guides/border-router/tools.md (hunk 3 above)
Comment (Vyrastas) [reply site/en/guides/border-router/tools.md]: Remove the &lt; and &gt; it should just be

pskc {commissioner-credential} {extpanid} {network-name}

File: site/en/guides/border-router/tools.md (hunk 4 above)
Comment (Vyrastas) [reply site/en/guides/border-router/tools.md]: This doesn't appear to have been done

steering-data [{length}] {joiner-id}

**Review by Vyrastas — APPROVED** [review]

**Comment by Vyrastas** [comment]
Please double-check the Copybara import results but I think we're good to go here.

**Review by jwhui — APPROVED** [review]

**Pull request merged**